### PR TITLE
Milestone 5: Styling

### PR DESCRIPTION
**Pull request by ioilmio**
This milestone is about styling the bookstore using the provided design:
- used sass
- svg for the circle
- Montserrat and RobotoSlab as font-family
- converted most px to rem or vw for future responsiveness

### CODE REVIEW

File: src/styles/scss/book.scss [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    // // Additional resets for further consistency
+    // background-color: transparent;
+    // border: none;
+    // padding: 0 1em 0 0;
+    // margin: 0.5rem;
+    // // width: 100%;
+    // font-family: $Montserrat;
+    // font-size: inherit;
+    // cursor: pointer;
+    // line-height: inherit;
```
Comment (uche-inyama) [inline src/styles/scss/book.scss]: #### Remove these commented lines of code.

File: src/styles/scss/booklist.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@import "config";
+
+.booklist {
+  // display: flex;
+  // justify-content: center;
+  margin-top: 2rem;
+  // margin-bottom: 2rem;
+  // border: 1px solid red;
+  // width: 80%;
```
Comment (uche-inyama) [inline src/styles/scss/booklist.scss]: #### To make your codebase neat, remove these lines of code.

File: src/styles/scss/nav.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    // font-family: $Montserrat;
+    // font-size: 13px;
+    // font-weight: 700;
+    // font-stretch: normal;
+    // font-style: normal;
+    // line-height: normal;
+    // letter-spacing: 12px;
+    // color: #121212;
+    // opacity: 0.5;
+    // }
```
Comment (uche-inyama) [inline src/styles/scss/nav.scss:37]: #### Removed these commented lines of code.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi @ioilmio :wave: 

Your design is looking good. But you have left a handful of commented lines of code in your .scss files.

I have highlighted a few in the inline comments. Below are the list of files with one line comments:

*    nav.scss, line 52.
*    form.scss, line 20.
*    _config.scss, line 12.

Remove these comments from your codebase, then submit another form for a code review.

### Have a great day :slightly_smiling_face: 

#### Happy coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @ioilmio, great job on this project :+1: Having removed all the commented code in our files :smile: I hereby approved the project:+1: 

[Optional]
- The README file must include instructions about how to clone and run the project, tools or languages for the project, a live version, social handles of the author, etc

Kindly go ahead and merge into development and continue with your next projects :slightly_smiling_face: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**